com.unity.render-pipelines.high-definition/Editor/Material/Eye/ShaderGraph/EyeSubTarget.cs

```diff
@@ -27,7 +27,6 @@ sealed partial class EyeSubTarget : LightingSubTarget, ILegacyTarget, IRequiresD
         };
 
         protected override string[] templateMaterialDirectories => passTemplateMaterialDirectories;
-        protected override string customInspector => "Rendering.HighDefinition.LightingShaderGraphGUI";
```

Comment:
why is there a change in Images/TerrainPaintHoles.png (above, wasn't able to comment :) )

Reply:
Hum, probably a merge conflict, I'll revert it